andela/haven-ah-backend
# #161342820 Update comment model

#### What does this PR do?
Update comment model.

#### Description of Task to be completed
- generate migration to add column `hasBeenEdited` to comment model.
- set `hasBeenEdited` to true when edit operation has been performed.

#### How should this be manually tested?
- on the terminal run git clone https://github.com/andela/haven-ah-backend.git.
- run git checkout `bg-update-comment-model-161342820`.
- ensure that PostgreSQL is running on your local machine.
- set up databases (test and development) following the `.env.sample` file.
- run `npm install` to install the npm packages.
- run `npm test` to run the unit and integration tests.

To test on postman
- run `npm run start-dev` and open up postman.

#### Screenshots where applicable
<img width="1098" alt="screen shot 2018-12-18 at 11 02 38 am" src="https://user-images.githubusercontent.com/29674347/50146639-7c0b6080-02b4-11e9-9d9b-8d7818816709.png">

#### Relevant Pivotal Tracker stories
#161342820
